login-workflow/src/new-architecture/screens/ContactScreen/ContactScreenBase.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react';
+import { Divider, Typography } from '@mui/material';
+import { WorkflowCard, WorkflowCardActions, WorkflowCardBody, WorkflowCardHeader } from '../../components/WorkflowCard';
+import { ContactScreenProps } from './types';
+import Box, { BoxProps } from '@mui/material/Box';
+import { ContactScreenClassKey, getContactScreenUtilityClass } from './utilityClasses';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+/**
+ * Component renders a screen with contact information for support with the application.
+ * Contact information is pulled from the context passed into the workflow.
+ *
+ * @param title to display in the header
+ * @param icon to display in the icon
+ * @param description to display the contact support content
+ * @param contactEmail to display the contact email
+ * @param contactPhone to display contact phone number
+ * @param dismissButtonLabel to display label text in button
+ * @param onDismiss function to call when user clicks button
+ * @param slots used for each slot in `ContactScreen`
+ * @param slotProps applied to each slot
+ *
+ * @category Component
+ */
+
+const LinkStyles = {
+    fontWeight: 600,
+    textTransform: 'none',
+    textDecoration: 'none',
+    color: 'primary.main',
+    '&:visited': {
+        color: 'inherit',
```

Comment:
This CSS is not applying when making it **visited**.
<img width="1632" alt="Screenshot 2023-06-07 at 9 28 39 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/133877691/fd359c5f-8948-4576-b393-e7a1d5933801">

Reply:
I see it does not work as its setup and will write an issue to add link styles visited in our theme repo

Reply:
https://github.com/etn-ccis/blui-react-themes/issues/74